dts/x86/intel_curie.dtsi

```diff
@@ -136,5 +136,33 @@
 
 			status = "disabled";
 		};
+
+		spi0: spi@0xb0001000 {
+			compatible = "snps,designware-spi";
+			#address-cells = <1>;
+			#size-cells = <0>;
+			reg = <0xb0001000 0x400>;
+			label = "SPI_0";
+			interrupts = <2 IRQ_TYPE_LEVEL_HIGH 2>;
+			interrupt-parent = <&intc>;
+			interrupt-masq = <0xb0800454>;
+			fifo-depth = <7>;
+
+			status = "disabled";
+		};
+
+		spi1: spi@0xb0001400 {
+			compatible = "snps,designware-spi";
+			#address-cells = <1>;
+			#size-cells = <0>;
+			reg = <0xb0001400 0x400>;
+			label = "SPI_1";
+			interrupts = <3 IRQ_TYPE_LEVEL_HIGH 2>;
+			interrupt-parent = <&intc>;
+			interrupt-masq = <0xb0800458>;
```

Comment:
where is interrupt-masq defined or used (what binding specifies this?)

Reply:
there is a patch adding it.

Reply:
So interrupt-masq should be a second element in the reg property.  As this is describing a register region that needs access.  You can use 'reg-names' if you want  to have names for the properties as well.  So something like:

```
reg = <0xb0001400 0x400
            0xb0800458 0x4>;
reg-names = <"base" "irq-masq">;
```